gnovm/tests/files/fun10.gno

```diff
@@ -13,5 +13,5 @@ func main() {
 }
 
 // Output:
-// nil func()()
+// func()()
```

Comment:
I'm not sure this is desirable.  I think it is still valuable to know if the function is nil or not.